### ISSUE TITLE
Can't select a custom java path on linux

### ISSUE THREAD

**Issue by nddragoon** [killpowa/Twickt-Launcher#1186]
**Describe the bug**
I can't select a custom java path on linux. When i use the file manager to select `/usr/lib/jvm/java-11-openjdk-amd64/bin/java` it appears on GDLauncher as `/run/user/1000/doc/a2c281e4/java` and when running the instance it says "permission denied". When i manually type in the path in the text box it says "no such file or directory". 
I also tried to make a link to java 11 in `/usr/bin/` but got the exact same result

**Expected behavior**
Setting the java path should use that executable

**Screenshots**
![image](https://user-images.githubusercontent.com/54559451/144098203-06b84e17-1609-4e82-ae16-cff1c7c44aa6.png)
![image](https://user-images.githubusercontent.com/54559451/144098264-1c255659-6efb-496e-8dd9-d716225ac1ae.png)
![image](https://user-images.githubusercontent.com/54559451/144098362-a99bdc95-786f-447a-a36a-a58766d0a8ad.png)
![image](https://user-images.githubusercontent.com/54559451/144098393-23739ec4-740c-479e-9129-544558305b90.png)
![image](https://user-images.githubusercontent.com/54559451/144098515-a921c1ec-b65e-46b5-9e24-704b6a7889b8.png)

**Operating System:**
- OS: Linux Mint 20.2 Cinnamon
- Java Version: need 11, but happens with any other version too

**Comment by Eskaan** [comment]
Did you try to chmod the executable for execution?

**Comment by nddragoon** [comment]
yeah i tried both `chmod a+x` and `chmod a+r` like someone suggested on the discord, same results

**Comment by egut** [comment]
I get the same problem.

On linux system-default is `/usr/bin/java` and when adding that I get  `/run/user.... `similar as @nddragoon and it point to a fail that is the same as the the correct executable but in the wrong place. Why it follow ups links is unclear and unneeded. 

If I add `/usr/bin/java` in the field or the correct path to java it cant find it (No such file or directory ).

All pathes was tested on cli and are correct. The auto-download do work but not system java.

**Comment by thecow275** [comment]
it most likely is that the appimage wont be able to launch java as appimages by default arent allowed to execute anything else than what is included with the appimage or is in the running directory of the appimage and it is due to security concerns as linux is used on servers and we dont want someone to accidentally infect a server by running a malicious appimage they downloaded from some random website.
